cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -91,11 +91,7 @@ spec:
       hostNetwork: true
       serviceAccountName: kube-proxy
       tolerations:
-      - key: {{ .MasterTaintKey }}
-        effect: NoSchedule
-      - key: {{ .CloudTaintKey }}
-        value: "true"
-        effect: NoSchedule
+      - operator: Exists
```

Comment:
This seems like a sane default to me, however I'm wondering if we should allow for it to be overridden using the kubeadm config file. @timothysc thoughts?

Reply:
What would be the rational for adding an override?

I think this PR as is is fine and we can add an override later if it turns out we need it. I'm basing this statement on my understanding of kube-proxy and the documentation which explicitly says "The Kubernetes network proxy runs on each node".

I haven't seen very many snowflake clusters though, so I'm not sure how likely it is folks would want anything besides tolerating everything here.